### PR TITLE
Add `ee-jwt-claim-id` to the CLI options

### PR DESCRIPTION
**Pull request by StefanBratanov**
Add the new CLI option introduced in this PR: https://github.com/Consensys/teku/pull/7816

### CODE REVIEW

File: docs/reference/cli/index.md [hunk 1]
```diff
@@ -652,6 +652,41 @@ ee-jwt-secret-file: "ee-jwt-secret.hex"
...
+  <TabItem value="Configuration file" label="Configuration file" >
+
+```bash
+ee-jwt-claim-id: "foobar"
+```
+
+  </TabItem>
+</Tabs>
+
+A unique identifier for the consensus layer client. This identifier will be added to the JWT claims as an `id` claim when using the Engine JSON-RPC API.
```
Comment (mjsmike62) [inline docs/reference/cli/index.md:688]: Maybe the statement s/b... `ee-jwt-claim-id` is a unique identifier for the consensus layer client. When using the JSON-RPC API engine, this identifier is added to JWT claims as an `id` claim.

File: versioned_docs/version-23.12.1/reference/cli/index.md [hunk 2]
```diff
@@ -652,6 +652,41 @@ ee-jwt-secret-file: "ee-jwt-secret.hex"
...
+  <TabItem value="Configuration file" label="Configuration file" >
+
+```bash
+ee-jwt-claim-id: "foobar"
+```
+
+  </TabItem>
+</Tabs>
+
+A unique identifier for the consensus layer client. This identifier will be added to the JWT claims as an `id` claim when using the Engine JSON-RPC API.
```
Comment (mjsmike62) [inline versioned_docs/version-23.12.1/reference/cli/index.md:688]: Maybe the statement s/b... `ee-jwt-claim-id` is a unique identifier for the consensus layer client. When using the JSON-RPC API engine, this identifier is added to JWT claims as an `id` claim.

**Review by mjsmike62 — APPROVED** [review]
Reviewed with two suggested edits.

**Comment by StefanBratanov** [comment]
> Reviewed with two suggested edits.

Thank you. I like the suggested edit apart from the repetition of the variable in the first sentence. Changed accordingly. What do you think?

**Comment by mjsmike62** [comment]
@StefanBratanov, I'm okay with you not using the variable in the first sentence.

**Pull request merged**